python-sdk/example_dags/example_dataframe_api.py

```diff
@@ -41,7 +42,7 @@ def load_and_group_covid_data():
 
 
 @aql.dataframe(columns_names_capitalization="original")
-def find_worst_covid_month(dfs: List[pd.DataFrame]):
+def find_worst_covid_month(dfs: list[pd.DataFrame]):
```

Comment:
Did you have to make such changes explicitly or we were able to do this automatically?

Reply:
Looks like the test is failing only on this PR. It seems stable on other PRs and on main.

Reply:
I have rebased the PR, let's see :)